src/main/java/net/clgd/ccemux/emulation/CCEmuXAPI.java

```diff
@@ -107,6 +110,58 @@ public CCEmuXAPI(CCEmuX emu, EmulatedComputer computer, String name) {
 
 			return new Object[] {};
 		});
+
+		methods.put("attach", o -> {
+			if (o.length > 1 && o[0] instanceof String && o[1] instanceof String) {
```

Comment:
Minor nitpick, but `o.length >= 2` would be more clear to show how many arguments it expects. There are other instances of this that need to be changed, as well.